pkg/configurator/types.go

```diff
@@ -65,4 +65,7 @@ type Configurator interface {
 
 	// GetOutboundIPRangeExclusionList returns the list of IP ranges of the form x.x.x.x/y to exclude from outbound sidecar interception
 	GetOutboundIPRangeExclusionList() []string
+
+	// GetEnablePrivilegedInitContainer determines whether init containers should be privileged
+	GetEnablePrivilegedInitContainer() bool
```

Comment:
This function name could read better:
```suggestion
	IsPrivilegedInitContainer() bool
```